declarations/Binance.history.json

```
@@ -0,0 +1,18 @@
+{
+  "Terms of Service": {
+    "fetch": "https://www.binance.com/en/terms",
+    "select": [
+      "main"
+    ],
+    "executeClientScripts": true,
+    "validUntil": "2024-01-23T12:31:55.000Z"
+  },
+  "Privacy Policy": {
+    "fetch": "https://www.binance.com/en/privacy",
+    "select": [
+      "main"
+    ],
+    "executeClientScripts": true,
+    "validUntil": "2024-03-12T00:32:36.000Z"
+  }
+}
```

This is not a well formated history file ;)
```suggestion
{
  "Terms of Service": [
    {
      "fetch": "https://www.binance.com/en/terms",
      "select": [
        "main"
      ],
      "executeClientScripts": true,
      "validUntil": "2024-01-23T12:31:55.000Z"
    }
  ],
  "Privacy Policy": [
    {
      "fetch": "https://www.binance.com/en/privacy",
      "select": [
        "main"
      ],
      "executeClientScripts": true,
      "validUntil": "2024-03-12T00:32:36.000Z"
    }
  ]
}
```

---

looks like a copy+paste error when copying